ui/imports/shared/popups/CommunityIntroDialog.qml

```diff
@@ -14,6 +16,7 @@ StatusDialog {
 
     property string name
     property string introMessage
+    property int access
```

Comment:
I'd find "accessType" more descriptive name for this.

Reply:
fixed :)